### DISCUSSION TITLE
Mget() and fget()

### DISCUSSION THREAD

**Discussion by Pyetro-Silva** [nesbox/TIC-80#2266] [Q&A]
I'm having trouble doing collision in lua script using mget() and fget() need help

**Comment by Skeptim** [comment]
Hi,
I am not sure I'll be able to help you myself, but people are here to answer questions so you can develop your problem without asking first ^^
Note that the [discord](https://discord.gg/HwZDw7n4dN) may be more active and plenty of people willing to help others.
And there is also tutorials about collision, mget and fget [here](https://github.com/nesbox/TIC-80/wiki/tutorials) if that can help you.
